.github/workflows/packaging.yml

```diff
@@ -31,7 +31,7 @@ jobs:
       - name: Install Taipy without dependencies
         run: |
           pip install .
-          rm -r taipy
+          rm -rf src taipy
```

Comment:
Do we still need 'taipy'' at this point?

Reply:
we dont!